tests/phpunit/tests/general/template.php

```diff
@@ -41,6 +42,7 @@ public static function wpTearDownAfterClass() {
 	public function set_up() {
 		parent::set_up();
 
+		switch_theme( 'default' );
```

Comment:
```suggestion
		switch_theme( WP_DEFAULT_THEME );
```

Use constant here. 

🔢 Applies to a others too

Reply:
@mukeshpanchal27 Not sure about this. The `default` is hard-coded in several unit tests, and the `WP_DEFAULT_THEME` constant may have a different value. So it's safer to guarantee that `default` is used.